### ISSUE TITLE
Redo ratings system so that the video's watch ID is used instead of the internal ID

### ISSUE THREAD

**Issue by PF94** [squarebracket-dev/squarebracket#52]
This was one of the reasons which prevented me from migrating old PokTube content.

**Comment by PF94** [comment]
additional idea I had was to add a “counting id” which would be used to order videos. meaning that poktube videos would appear earlier in the site but would be listed later in the admin panel.

**Comment by PF94** [comment]
ethier that or videos are sorted by upload date, which could sort stuff inaccurately if two videos share the same timestamp

**Comment by PF94** [comment]
ok i honestly don't know the fuck i meant a year ago

poktube migration originally was planned to refresh all video's internal ID numbers due to squarebracket originally ordering videos via the ID and not upload date. so let's say if the fifth video was ID 5, and then there were 50 poktube videos migrated over. this would mean that the fifth squarebracket video would have been jumped to ID 55. i think my issue with this was that ratings wouldn't be moved due to it using the internal ID and not the random ID.

in conclusion, if we went with the Randley layout's weird attempt at trying to imitate e6 but with Finalium SCSS (WHY) and the weird "unified content" system which dropped random IDs (tf????), this would have been a problem, but it isn't the case as we're still going to use random IDs.

however poktube videos before about the middle of march 2021 did not properly save the time a video was uploaded, only the date (we didn't even use unix timestamps what the fuck), so ordering will be fucked for poktube videos.
